utils/controllerFactory.js

```diff
@@ -62,19 +61,18 @@ module.exports = function controllerFactory(
 
   sinkData.forEach(({ method, uri, sink, key }) => {
     router[method](`${uri}/safe`, async (req, res, next) => {
-      const { input } = get(req, key);
+      const input = utils.getInput({ model: locals, req, key });
```

Comment:
can we change the signature of getInput to take `locals` rather than   `model` here? Kraken is the only application with the concept of 'models' currently, i think this is just a little bit confusing.

in test-bench-utils:
```js
module.exports.getInput = function getInput({ model, locals = model, req, key }) {
   return locals.input || get(req, key).input;
 };
```